packages/runtime/src/api/react.tsx

```diff
@@ -60,6 +60,9 @@ const typePolicies: TypePolicies = {
       table(existingData, { args, toReference }) {
         return existingData ?? toReference({ __typename: 'Table', id: args?.id }, true)
       },
+      page(existingData, { args, toReference }) {
+        return existingData ?? toReference({ __typename: 'Page', id: args?.id }, true)
+      },
```

Comment:
Do we need a type policy for snippets as well?

Reply:
We don't need it right now because we don't query the snippets directly.